src/bindings/python/requirements_test.txt

```diff
@@ -29,7 +29,7 @@ flake8_quotes
 mypy
 Pep8-naming
 pydocstyle
-pytest-forked
+pytest-forked; platform_system != "Windows"
```

Comment:
we have use of different markers in different places. Here you use `platform_system != "Windows"` and in MO we use `sys_platform == 'win32'`. May be it makes sense to align them?

Reply:
`sys_platform == 'win32` seems to be a lot more common in our repository, I removed `platform_system`.